src/index.html

```diff
@@ -1,12 +1,414 @@
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="page__catalog catalog">
+      <div class="catalog__container">
+        <div class="catalog__card card"
+          data-qa="card"
+        >
+          <img
+            src="./images/imac.jpeg"
+            alt="Imac"
+            class="card__img"
+          >
+          <h3 class="card__title">
+            Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+          </h3>
+          <p class="card__code">
+            Код товара: 195434
+          </p>
+          <div class="card__rating">
+            <div class="card__stars stars stars--4">
+              <div class="stars__star star"></div>
+              <div class="stars__star star"></div>
+              <div class="stars__star star"></div>
+              <div class="stars__star star"></div>
+              <div class="stars__star star"></div>
+            </div>
+            <p class="card__review">
+              Отзывов: 5
+            </p>
+          </div>
+          <div class="card__value">
+            <p class="card__price">
+              Цена:
+            </p>
+            <p class="card__amount">
+              69 999 грн
+            </p>
+          </div>
+          <a
+            href="#"
+            class="card__button"
+            data-qa="card-hover"
+          >
+            Купить
+          </a>
+        </div>
+
+        <div class="catalog__card card">
+          <img
+            src="./images/imac.jpeg"
+            alt="Imac"
+            class="card__img"
+          >
+          <h3 class="card__title">
+            Моноблок APPLE A1419 iMac 27&quot; Retina 5K (MNED2UA/A)
+          </h3>
+          <p class="card__code">
+            Код товара: 195434
+          </p>
+          <div class="card__rating">
+            <div class="card__stars stars stars--4">
+              <div class="stars__star star"></div>
+              <div class="stars__star star"></div>
+              <div class="stars__star star"></div>
+              <div class="stars__star star"></div>
+              <div class="stars__star star"></div>
+            </div>
+            <p class="card__review">
+              Отзывов: 5
+            </p>
+          </div>
+          <div class="card__value">
+            <p class="card__price">
+              Цена:
+            </p>
+            <p class="card__amount">
+              69 999 грн
+            </p>
+          </div>
+          <a
+            href="#"
+            class="card__button"
+          >
+            Купить
+          </a>
+        </div>
+
+        <div class="catalog__card card">
+          <img
+            src="./images/imac.jpeg"
+            alt="Imac"
+            class="card__img"
+          >
+          <h3 class="card__title">
+            Моноблок APPLE A1419 iMac 27&quot; Retina 5K (MNED2UA/A)
+          </h3>
+          <p class="card__code">
+            Код товара: 195434
+          </p>
+          <div class="card__rating">
```

Comment:
```suggestion
          </h3>
          
          <p class="card__code">
            Код товара: 195434
          </p>
          
          <div class="card__rating">
```

Check list №2 [CODE STYLE] - Add empty lines between multiline sibling blocks of HTML. But don't add empty lines between parent and child elements
у тебя там такое - используй пробелы)